tiled/catalog/adapter.py

```diff
@@ -1330,6 +1330,7 @@ def in_memory(
         access_policy=access_policy,
         writable_storage=writable_storage,
         readable_storage=readable_storage,
+        init_if_not_exists=True,
```

Comment:
This is not necessary. When the database is in memory, the tables are _always_ initialized because by definition this is a new database in process memory. This keyword argument has the effect of spawning a subprocess to initialize a database, which cannot help in this case.

Reply:
Removed. Oops.